### ISSUE TITLE
Hide pseudo-content from SR

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#854]
Part of #820 

Some of the characters we use as separators and other visual indicators are read by screen readers in unintended ways.

» reads as "double-angle quotation mark"
> Indiana University, Erminie Wheeler-Voegelin Archives » Great Lakes - Ohio Valley Ethnohistory Collection, 1953-1966 » Tribal History Documents Series, 1540-1957 » Potawatomi, 1600-1898 » Potawatomi, 1726-1749 » Potawatomi, 1733-1749

- [x] Wrap the `breadcrumb_separator` in `<span aria-hidden="true"> ... </span>` to hide it from the SR

And just to record, no action required:
⟩   `"\027E9"` reads as "mathematical right angle bracket." All of these have been or will be replaced by svgs.
